backend/test/NightStateChangeEvents.test.js

```diff
@@ -6,22 +6,23 @@ const roles = require('../domain/Enum/Role');
 
 describe('start-night unit tests', () => {
     const port = process.env.PORT || config.local_port;
+    console.log("File 3")
```

Comment:
Are we keeping console logs?

Reply:
nope it was an error thanks for catching